.github/ISSUE_TEMPLATE/Suggestion.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Suggestion
+about: Suggest new features or an idea for OSORI
+title: ''
+labels: review required
```

Comment:
```suggestion
labels: suggestion, review required
```
What about adding a label "suggestion" for this type of issue registration?

Reply:
Done